iib/workers/tasks/build.py

```diff
@@ -731,14 +741,19 @@ def handle_rm_request(
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
     output_pull_spec = _create_and_push_manifest_list(request_id, arches)
 
-    _finish_request_post_build(
+    _update_index_image_pull_spec(
         output_pull_spec,
         request_id,
         arches,
         from_index,
         overwrite_from_index,
         overwrite_from_index_token,
     )
+    set_request_state(
+        request_id,
+        'complete',
+        'Operator bundle(s) removed from index image, completing the request successfully',
```

Comment:
In this case, it's not individual bundles that are removed, it's all bundles associated with an operator. Also, I don't think there is any value in adding `, completing the request successfully`. The word `completing` insinuates there is more that will occur but actually, the request is complete when the state is set to `complete`.
```suggestion
        'The operators were successfully removed from index image',
```

Reply:
same as earlier, I actually liked this way of message